pandas/tseries/resample.py

```diff
@@ -686,6 +686,10 @@ def _downsample(self, how, **kwargs):
         if not len(ax):
             # reset to the new freq
             obj = obj.copy()
+            if how == "size" and isinstance(obj, pd.DataFrame):
```

Comment:
@jreback the `size` method of resampler of the empty dataframe must always return the empty series for each frequency, doesn't it?